swagger.yaml

```diff
@@ -5692,9 +5696,11 @@ definitions:
         type: string
         description: Boot device to write image on
       controller_image:
+        pattern: '^(([a-zA-Z0-9\-\.]+)(:[0-9]+)\/)?[a-z0-9\._\-\/@]+:[a-zA-Z0-9_\-.]+$'
```

Comment:
Seems that this does not allow image without tag, although it should be valid.

@avishayt what do you think ?

Reply:
The rule of thumb should be that we allow all legal configurations (so we don't shoot ourselves in the foot) while blocking illegal configurations (for security reasons).